shell/platform/windows/flutter_windows_view_unittests.cc

```diff
@@ -665,17 +665,11 @@ TEST(FlutterWindowsViewTest, CheckboxNativeState) {
       FlutterSemanticsFlag::kFlutterSemanticsFlagHasCheckedState);
   bridge->AddFlutterSemanticsNodeUpdate(&root);
   bridge->CommitUpdates();
-  root_node = bridge
-                  ->GetFlutterPlatformNodeDelegateFromID(
-                      AccessibilityBridge::kRootNodeId)
-                  .lock();
```

Comment:
Here and below -- you're 100% sure the root node (and the associated IAccessible) are never replaced with a new object? 

My recollection is that that's true, but it's been well over a year since I looked. Alternatively we could scope the individual tests:
```cc
{
  // check foo.
  auto root_node = bridge->...
  EXPECT(...)
}

{
  // check bar.
  auto root_node = bridge->...
  EXPECT(...)
}
```